Sources/Core/GTMSessionFetcherLogging.m

```diff
@@ -385,8 +388,8 @@ - (void)logFetchWithError:(NSError *)error {
   //
   // we'll use a local variable since this routine may be reentered while waiting for formatting
   // to be completed.
-  static int gResponseCounter = 0;
-  int responseCounter = ++gResponseCounter;
+  static atomic_int gResponseCounter = 0;
+  int responseCounter = atomic_fetch_add(&gResponseCounter, 1);
```

Comment:
++gResponseCounter should be sufficient here https://en.cppreference.com/w/c/language/atomic.html

Reply:
Done